sdk/netapp/azure-resourcemanager-netapp/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-02-18)
+
+- Azure Resource Manager NetApp client library for Java. This package contains Microsoft Azure SDK for NetApp Management SDK. Microsoft NetApp Azure Resource Provider specification. Package tag package-netapp-2020-11-01. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
Please use the term `Azure NetApp Files` in place of just `NetApp` as that is the official marketing term and someone will complain if they happen to see this.